### ISSUE TITLE
CodeFileProblem name does not exist on task_problems

### ISSUE THREAD

**Issue by amrondonp** [UCL-INGI/INGInious#314]
https://github.com/UCL-INGI/INGInious/search?utf8=%E2%9C%93&q=CodeFileProblem&type=
Learning how to use the subproblems plugins. I saw that this file is making reference to a class that does not exists. I have not sent to you a PR changing it to FileProblem because I do not know if this file is actually being used or if you plan to delete it someday

![image](https://user-images.githubusercontent.com/6508705/38071672-47a8be2e-32e8-11e8-93ad-364131e043dd.png)

**Comment by anthonygego** [comment]
Indeed. Gonna fix it. This file is used with pybabel for extracting translatable strings from yaml files to gettext pot files.
